test/stubString.js

```diff
@@ -0,0 +1,11 @@
+import * as RA from '../src/index';
+import eq from './shared/eq';
+
+describe('stubString', function() {
+  it("tests `function` that returns `'''`", function() {
+    eq(RA.stubString(), '');
+    eq(RA.stubString([1]), '');
+    eq(RA.stubString(new Array()), '');
+    eq(RA.stubString(1, 2, 3), '');
+  });
```

Comment:
context('when called', function() {
  specify('should always return empty string', function() {
 ....test here for equality of results of 2 consequent invocations
  });
});